athletic_pandas/algorithms/__init__.py

```diff
@@ -11,6 +11,20 @@
 
 
 def mean_max_power(power):
+    """Mean-Max Power
+
+    Also known as a Power-Duration-Curve
+
+    Parameters
+    ----------
+    power: pd.Series
+
+    Returns
+    -------
+    pd.Series
+
+    TO-DO: to be replaced by vmpy.algorithms.power_duration_curve()
```

Comment:
It does, I've moved it to vmpy.metrics.power_duration_curve() the last moment. It calculates accumulated energy first and than does two arrays diff with a shift. It speeds up calculations by x4. The 8hours ride takes about 10 seconds instead of almost a minute on my PC.

Reply:
Oh fancy! Did you benchmark it to my implementation or yours?